src/common/KBaseButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { FunctionComponent as FC, MouseEventHandler } from 'react';
+import classes from './KBaseButton.module.scss';
+
+interface KBaseComponentProps {
+  disabled?: boolean;
+  onclick?: MouseEventHandler;
+}
+
+export const KBaseButton: FC<KBaseComponentProps> = ({
+  children,
+  disabled,
+  onclick,
+}) => {
+  return (
+    <button className={classes.button} onClick={onclick} disabled={disabled}>
+      {children}
+    </button>
+  );
+};
```

Comment:
```suggestion
export const KBaseButton: FC<React.HTMLProps<HTMLButtonElement>> = (props) => {
  return (
    <button {...props} className={[classes.button, props.className].join(' ')}>
      {props.children}
    </button>
  );
};
```

Reply:
This was almost perfect advice, but the type ended up being `FC<React.HTMLAttributes<HTMLButtonElement>>` after running into [this issue](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/36505).